### PR TITLE
Auto-Inscribe Markdown Posts on 100 Sats for RelayOne Users

### PR DESCRIPTION
**Pull request by stevenzeiler**
The important logic here is under `services/inscriptions.js` and `components/MarkdownComposer.tsx`

Once this is merged and deployed to the markdown composer and we see live results it might need to be tweaked or perhaps we can roll out this feature to all ways that people compose content on pow.co.

For Relay users it is by default inscribing to a satoshis owned by the `run owner`. For sensilet it can use the normal sensilet public key hash address, and for handcash it ought to use the static handcash receiving address.

### CODE REVIEW

**Comment by 1aristotelis** [comment]
Why using the `MarkdownComposer` and not the `SocialEditor`  (which is now the default editor)?
Unless the user willingly type https://pow.co/compose/markdown he won't access this feature

**Comment by stevenzeiler** [comment]
I prefer the markdown editor and have used both. My preference would be to re-add the button that allows the option for full markdown editor, and I know that preference is shared by others who vocalized it at the meetings.

**Comment by stevenzeiler** [comment]
> Why using the `MarkdownComposer` and not the `SocialEditor` (which is now the default editor)? Unless the user willingly type https://pow.co/compose/markdown he won't access this feature

Also the idea with using MarkdownComposer is exactly because if there are any issues with the new feature nobody will be impacted unless they are really trying to use it.

**Comment by stevenzeiler** [comment]
Once the concept is proven to work in production on MarkdownComposer I would like to roll it out to all places where users share content.

**Comment by 1aristotelis** [comment]
Shall we give the user the option to inscribe their post ? I'm not sure I'm willing to inscribe all of my posts

**Comment by stevenzeiler** [comment]
> Shall we give the user the option to inscribe their post ? I'm not sure I'm willing to inscribe all of my posts

Certainly you are already writing your post into the blockchain, just not on top of some satoshis.

Yes the user could certainly be able to configure whether they want to inscribe and under which conditions.

If a player has not opted in to inscriptions onto their own satoshis I would like all posts to auto-inscribe onto satoshis sent to a wallet address controlled by pow.co.

That way pow.co still generates and may accumulate ordinals assets on every post while respecting the privacy of players when they choose not to associate their identity to content.

**Comment by stevenzeiler** [comment]
Perhaps inscriptions on by default is too intrusive, and an option should be added to opt-in to the feature, now before merging?

**Review by 1aristotelis — APPROVED** [review]
Let's merge as is to test and validate the feature. We'll improve on the next iteration

File: components/MarkdownComposer.tsx
Comment (usselman) [inline components/MarkdownComposer.tsx:1]: Epic

File: services/inscriptions.js
Comment (usselman) [inline services/inscriptions.js:1]: very interesting!

**Review by usselman — APPROVED** [review]
A cool update. I am excited to see where this leads.

**Pull request merged**